api/src/schema/index.js

```diff
@@ -103,7 +111,7 @@ const Schema = i18n => {
     type Query {
       # ${i18n.t`Details for a specific dwelling`}
       evaluationsFor(account: Int!): Dwelling
-      dwellingsInFSA(filter: Filter forwardSortationArea: ForwardSortationArea!): [Dwelling]
+      dwellingsInFSA(filter: Filter forwardSortationArea: ForwardSortationArea! limit: Int, next: String): PaginatedResultSet
```

Comment:
Couple quick questions:

1. `limit: Int,` has a trailing comma but none of the others do. Is this intentional/meaningful? 
2. We don't care about having a `previous` parameter, eh?

Reply:
Regarding the comma, GraphQL has what it calls [insignificant commas](http://facebook.github.io/graphql/October2016/#sec-Insignificant-Commas). I normally don't put them for that reason, but clearly I had a lapse in consistency.

There is definitely some more testing that needs to be added to fully exercise the pagination stuff. Adding a previous parameter should happen for sure... maybe that would be a good time to remove that unsightly comma.

Reply:
👍 for mvp pagination